card/spec/lib/card/view_cache_spec.rb

```diff
@@ -1,3 +1,5 @@
+# -*- encoding : utf-8 -*-
```

Comment:
I get a bunch of "non-ascii character- default to utf-8" warnings each time I run the tests. Hoped that helped but it didn't.